EIPS/eip-3675.md

```diff
@@ -189,6 +197,9 @@ After the upgrade of the consensus mechanism only the beacon chain network will
 
 It is recommended for the client software to not propagate descendants of any terminal PoW block to reduce the load on processing the P2P component and stop operating in the environment with unknown security properties.
 
+### Restricting the length of `extraData`
+
+The `extraData` field has been specified as unbounded in the PoW spec even though mainnet and most PoW testnets only utilize `32` bytes for it.  `extraData` fields of greater length are used by clique testnets and other networks to carry special signature/consensus schemes. This EIP restricts the length of `extraData` to `32` bytes since any network that is transitioning from another consensus mechanism to a beacon chain PoS consensus mechanism no longer needs extended or unbounded `extraData`.
```

Comment:
The yellow paper (which defines ethereum pow/mainnet consensus) notes that "... extraData must be at most
32 bytes." Thus it is not unbounded for PoW network spec as noted in the original comment.

Suggest using the following text (or similar):

```suggestion
The `extraData` field is defined as a maximum of `32` bytes in the yellow paper. Thus mainnet and most PoW testnets cap the value at `32` bytes.  `extraData` fields of greater length are used by clique testnets and other networks to carry special signature/consensus schemes. This EIP restricts the length of `extraData` to `32` bytes because any network that is transitioning from another consensus mechanism to a beacon chain PoS consensus mechanism no longer needs extended or unbounded `extraData`.
```

Reply:
Thanks for the suggestion. Adopted in db1c213.